src/images/title.svg

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="762px" height="355px" viewBox="0 0 762 355" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>646B4B12-CAE8-49AE-962A-EBD8CE33CEB0</title>
```

Comment:
Shouldn’t the contents of an SVG `title` be readable text, for SEO and accessibility? This looks like a generated ID.

Reply:
Good catch. This was removed in a recent commit.